dev/AnimatedIcon/AnimatedIcon.cpp

```diff
@@ -165,12 +165,11 @@ winrt::Size AnimatedIcon::ArrangeOverride(winrt::Size const& finalSize)
             std::min(finalSize.Height / scale.y, visualSize.y)
         };
         const auto offset = (finalSize - (visualSize * scale)) / 2;
-        const auto z = 0.0F;
         const auto rootVisual = visual.RootVisual();
-        rootVisual.Offset({ offset, z });
+        rootVisual.Offset({ offset, 0.0f });
         rootVisual.Size(arrangedSize);
-        rootVisual.Scale({ scale, z });
-        return finalSize;
+        rootVisual.Scale({ scale, 1.0f });
```

Comment:
This seems unrelated to the change ? Interntional ?

Reply:
Intentional but not needed here. I could break it out to a separate change if you'd like